### DISCUSSION TITLE
"Play now" in one-click

### DISCUSSION THREAD

**Discussion by RayLation** [music-assistant/support#1549] [Q&A] [answered]
Is there any way to just make it "play now" when I click on a track/station/playlist/etc instead of having to click again on "play now" in the popup menu? I understand I can use this menu to choose the player, but it's always the same one in my case.

**Comment by OzGav** [comment]
Not at the moment although the UI is currently in active development. Note that the pop up also contains other options not just the player selection.

**Reply by OzGav** [reply, marked as answer]
This functionality is now available in the UI
